components/collator/src/elements.rs

```diff
@@ -20,11 +20,12 @@
 
 use core::char::REPLACEMENT_CHARACTER;
 use icu_collections::char16trie::TrieResult;
-use icu_collections::codepointtrie::CodePointTrie;
+use icu_collections::codepointtrie::CodePointTrieBorrow;
```

Comment:
Suggestion: CodePointTrieBorrowed for consistency

Reply:
Fixed.